example_test.go

```diff
@@ -92,3 +93,64 @@ func ExampleAppendInto() {
 	// call 3 failed
 	// foo; baz
 }
+
+func ExampleAppendInvoke() {
+	var err error
+
+	var errFunc1 multierr.Invoker = multierr.Invoke(func() error {
+		fmt.Println("call 1 failed")
+		return errors.New("foo")
+	})
+
+	var errFunc2 multierr.Invoker = multierr.Invoke(func() error {
+		fmt.Println("call 2 did not fail")
+		return nil
+	})
+
+	var errFunc3 multierr.Invoker = multierr.Invoke(func() error {
+		fmt.Println("call 3 failed")
+		return errors.New("baz")
+	})
+
+	defer func() {
+		fmt.Println(err)
+	}()
```

Comment:
optional: I wonder if we should do a full file example instead, as it may be easier to see the print from a separate function that calls a function using `AppendInvoke`. It's a little easier to think of this as the result the caller sees IMO

Reply:
you mean moving this to a new file only containing this very example and refactoring the anonymous function to normal functions?